services/order.py

```diff
@@ -0,0 +1,52 @@
+from django.db import transaction
+from django.utils import timezone
+from django.core.exceptions import ObjectDoesNotExist
+from db.models import Order, Ticket, MovieSession, User
```

Comment:
It's recommended to use `get_user_model()` instead of directly importing `User`. This aligns with the checklist item: 'Use `get_user_model()` instead of `User`, it is the best practice.'